hspec-discover/package.yaml

```diff
@@ -47,3 +47,6 @@ tests:
       - hspec-discover
       - hspec-meta >= 2.3.2
       - QuickCheck >= 2.7
+    verbatim: |
+      build-tool-depends:
+          hspec-meta:hspec-meta-discover
```

Comment:
It is more robust to use:

```
verbatim:
  build-tool-depends: ...
```
Specifically use a mapping instead of a string for `verbatim`.  A string will just get included literally in the generated `.cabal` file, on the other hand with a mapping `build-tool-depends` overrides any existing fields with the same name.  This is important as `hpack-0.30.0` will add `build-tool-depends` in certain situations.